scrooge-generator/src/main/resources/nodegen/service.mustache

```diff
@@ -92,10 +92,10 @@ export class Processor {
         this._handler = handler
     }
 
-    public process(input: TProtocol, output: TProtocol) {
+    public process(input: TProtocol, output: TProtocol, context?: any) {
```

Comment:
It's a great idea :) it'll make integration with our various server implementations much nicer